CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 6.0.0
+### NOTICE 
+
+## BREAKING CHANGES
+
+### Manual Deployment Step
+When upgrading from version 5.20 a manual step is required. Before deploying 6.0 delete both Azure App Functions and the Azure App Service plan before upgrading. This is required because we have migrated the service from `python` to `C#`. 
+
+After deployment, there will be two App Functions deployed, one with the name of the deployment and a second one with the same name and a `-net` suffix. This is a temporary situation and the `-net` app function will be removed in a following release. 
+
+If you have not used the deployment parameters to deploy C# functions in 5.20, you can manually delete the `-net` app function immediately. Deploying the C# functions was not a default action in 5.20, for most deployments deleting the `-net` app function immediately is ok. 
+
+### Deprecation of jinja templates
+With this release we are moving from jinja templates to [scriban](https://github.com/scriban/scriban) templates. See the documentation for [scriban here](https://github.com/scriban/scriban/tree/master/doc). 
+
+Version 6.0 will convert jinja templates on-the-fly for a short period of time. We do **_not_** guarantee that this will be successful for all jinja template options. These on-the-fly conversions are not persisted in the notifications table in this release. They will be in a following release. This will allow time for conversions of templates that are not handled by the current automatic conversion process. 
+
+
+### Added
+* Agent: Making machine identity a parameter of the agent config. [#2649](https://github.com/microsoft/onefuzz/pull/2649)
+### Changed
+* Service: Deployment enables refactored C# App Function. [#2650](https://github.com/microsoft/onefuzz/pull/2650)
+* CLI: Use broker or browser login instead of device flow. [#2612](https://github.com/microsoft/onefuzz/pull/2612)
+* Service: Update to .NET 7. [#2615](https://github.com/microsoft/onefuzz/pull/2615)
+* Service: Make Proxy `TelemetryKey` optional. [#2619](https://github.com/microsoft/onefuzz/pull/2619)
+* Service: Update OMI to 1.6.10.2 on Ubuntu VMs. [#2629](https://github.com/microsoft/onefuzz/pull/2629)
+* CLI: Make the `--container_type` parameter required. [#2631](https://github.com/microsoft/onefuzz/pull/2631)
+* Service: Improve logging around notification failures. [#2653](https://github.com/microsoft/onefuzz/pull/2653)
+### Fixed
+* Service: Notification Template `targetUrl` parameter fix. [#2625](https://github.com/microsoft/onefuzz/pull/2625)
```

Comment:
```suggestion
* Service: Notification Template `targetUrl` parameter fix. Only use the filename instead of the absolute path in the URL. The makes the links created in ADO bugs work as expected. [#2625](https://github.com/microsoft/onefuzz/pull/2625)
```